examples/chat/README.md

```diff
@@ -19,15 +19,15 @@ The first client to run typically creates the Oak Application and a first chat
 room inside it:
 
 ```bash
-$ ./examples/chat/run -C --room_name=MyRoom
+$ ./examples/chat/run
```

Comment:
See other comment, I think we should either add a `--dry-run` flag, or take this out of the `run_examples` script, as it's really more of an interactive thing anyways.

Reply:
Yeah, the invocation via `run_examples` was pretty minimal, but it did at least ensure that the Node code was still building and the frontend Node was able to be loaded and its `oak_main` function executed &ndash; I'd be loathe to lose that. 

How would a `--dry_run` flag work with `run_examples`?

Reply:
`--dry_run` (not the best name admittedly) would load the code to the oak runtime, and then either not interact with it at all, or interact minimally from the client point of view. 

Note that when #431 is done we could just load the application and check that it starts up fine (i.e. the server does not immediately crash), which would bring us back on par to what we have now.

Anyway let's discuss that in #458 to avoid this PR becoming even larger?

Reply:
Sorry, I wasn't clear about my question about `run_examples`: how would `run_examples` pass `--dry_run` to *just* this example, without also passing an unknown flag to all the other examples?  Are you expecting to add a special case in `run_examples` that watches out for this specific example?

BTW I've fixed #458, so I think something is needed here before this PR would be mergeable.

Reply:
Merged and temporarily removed the chat example from `run_examples`, and created #462 to track migration of examples to use the single-app server, which should make it easier to at least make sure that they all compile and start up correctly (even if they cannot be interacted with).